app/templates/about.jade

```diff
@@ -359,6 +359,16 @@ block content
                  p.small CodeCombat is looking for an experienced engineering leader to help build our next-gen learning platform as well as guide our friendly and efficient engineering team.
                  a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/38d91038-8417-4aa1-9936-77a082ffa050?lever-origin=applied&lever-source%5B%5D=CodeCombat" rel="external")
                   span(data-i18n="about.learn_more")
+            .job-listing
```

Comment:
Similar to other pull request comment. Could you add the line
`.col-sm-6.col-md-5.col-md-offset-1.col-lg-4.col-lg-offset-0`
above this line similar to the Eng Manager job position?